prestoadmin/util/slider.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Utility functions for dealing with slider
+"""
+
+
+def degarbage_json(garbage_output):
+    nest_level = 0
```

Comment:
I like that option better.  python will give you uniquely named temporary files, so that shouldn't be a problem.  And I don't think this would be called frequently enough to really worry about filling up the tmp directory if it crashes.  It would just be one stray file left behind in rare circumstances, and in a tmp directory.  
But I believe we could also instead do something like 'bin/slider status -out > cat' to print the output to stdout instead of saving it to a file.